include/arch/riscv/arch/32/mode/hardware.h

```diff
@@ -69,40 +69,38 @@
 
 static inline uint64_t riscv_read_time(void)
 {
-    uint32_t nH1, nL, nH2;
+    uint32_t nH_prev, nH, nL;
```

Comment:
what is the reasoning behind the changes in this commit?

Reply:
The variable names are more intuitive. Can also keep the old names.

Reply:
Changing the names makes the code new for anyone who understood it before. 1, 2 vs prev is not really intuitive, just a personal preference. To avoid code churn we should avoid renames like this.